packages/h5p-server/src/renderers/theme.ts

```diff
@@ -0,0 +1,772 @@
+import { ITheme } from '../types';
```

Comment:
The stylesheet replicates a lot of the CSS properties of the styles. I think it should only include overrides and avoid duplicating CSS properties that are already set by H5P.

I still think that it would be sensible to use SCSS here.

Reply:
It would also allow scenarios in which admins change the override SCSS and don't have to touch code.